src/planscape/templates/invites/new_invite_message.html

```diff
@@ -84,11 +84,22 @@
     <![endif]-->
     <div style="width: 100%; padding: 10px; color: black;">
       <span style="font-size: 22px; font-weight: 500; line-height: 32px">{{ inviter.get_full_name }}</span><span
-        style="font-size: 22px; font-weight: 600"> </span><span style="font-size: 22px; font-weight: 500">invited you
-        to</span><span style="font-size: 22px; font-weight: 600"> </span><span
-        style="font-size: 22px; font-weight: 700">collaborate</span><span style="font-size: 22px; font-weight: 600">
-      </span><span style="font-size: 22px; font-weight: 500">on </span><span
-        style="font-size: 22px; font-weight: 400">“{{ planning_area.name }}”</span>
+        style="font-size: 22px; font-weight: 600"> </span>
+      {% if role == 'VIEWER' %}
+      <span style="font-size: 22px; font-weight: 500">invited you to be a </span>
+      <span
+      style="font-size: 22px; font-weight: 700">viewer</span><span style="font-size: 22px; font-weight: 600">
+      {% elif role == 'COLLABORATOR' %}
```

Comment:
can we simplify this and just print the role (or a formatted version of the role) rather than repeating the lines for each?